canopen/source/co_sync.c

```diff
@@ -57,6 +62,7 @@ void COSyncInit(CO_SYNC *sync, struct CO_NODE_T *node)
     if (err != CO_ERR_NONE) {
         node->Error = CO_ERR_CFG_1005_0;
```

Comment:
I think we should remove this error, because this line forces all users to use synchronized PDO transfers. An object dictionary without 1005h should be possible (e.g. when SYNC is not used).

Setting the member `sync->CobId` to 0 is sufficient.

Reply:
You are right, fixed.

Reply:
I think that `CO_ERR_CFG_1005_0` will have its place in case of `COTPdoReset()` detects some PDO that is configured as synchronous but SYNC COB-ID is not defined, so i will keep it in `CO_ERR` enumeration.